python/ratbagd/__init__.py

```diff
@@ -35,13 +35,20 @@ class RatbagdDBusUnavailable(BaseException):
 class _RatbagdDBus(object):
     def __init__(self, interface, object_path):
         self._dbus = Gio.bus_get_sync(Gio.BusType.SYSTEM, None)
-        self._proxy = Gio.DBusProxy.new_sync(self._dbus,
-                                             Gio.DBusProxyFlags.NONE,
-                                             None,
-                                             'org.freedesktop.ratbag1',
-                                             object_path,
-                                             'org.freedesktop.ratbag1.{}'.format(interface),
-                                             None)
+        if self._dbus == None:
```

Comment:
Right, `is` checks for address equivalence and `==` checks for value equivalence.